app/Http/Controllers/ProjectsController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Project;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Input;
+use App\Http\Requests;
+
+class ProjectsController extends Controller
+{
+    public function index(){
+        $projects = Project::all();
+        return $projects;
+    }
+
+    public function show($id){
+        $project = Project::findOrfail($id);
```

Comment:
I think this should be handled if the record retrieval process failed by the findOrfail method